File: src/Clients/FrontOffice.Client/front-office/src/app/models/shopping/shopping.cart.ts
import { Guid } from "guid-ts";
import { ShoppingCartItem } from "./shopping.cart.item";

export class ShoppingCart {
  userId: Guid;
  items: Array<ShoppingCartItem>;
  count: number;
  totalPrice: number;
  constructor(userId: Guid, items: Array<ShoppingCartItem>) {
    this.userId = userId;
    this.items = items;
  }
}

File: src/Services/Basket/Basket.API/Models/ShoppingCart.cs
using System;
using System.Collections.Generic;

namespace Basket.API.Models
{
    public class ShoppingCart
    {
        public Guid UserId { get; set; }
        public List<ShoppingCartItem> Items { get; set; }
        public int Count { get; set; }
        public decimal TotalPrice { get; set; }
    }
}
